internal/net-mocks/example/vitest.setup.ts

```diff
@@ -0,0 +1,9 @@
+import { beforeAll } from "vitest";
+import { net } from "../src";
+
+beforeAll(() =>
+  net.setupVitest({
+    redactedKeys: ["x-api-key"],
```

Comment:
_question_: I wonder if we should allow specifying values rather than keys to be omitted? Also I wonder if we should auto-include certain keys or values such as OAI keys etc.